### ISSUE TITLE
Splash Screen is not centered on active monitor

### ISSUE THREAD

**Issue by MBach** [a85/POSTMan-Chrome-Extension#4412]
## App Details:
```
Postman for Linux
Version 6.0.9
linux 4.15.0-13-generic / x64
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there: IDK
 2. Expected behaviour: the splash screen should be centered on the default monitor (first launch) or latest one if already launched
 3. Console logs (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/ for the Chrome App, View->Toggle Dev Tools for the Mac app): N/A
 4. Screenshots
![bug](https://user-images.githubusercontent.com/1532657/38253625-695f9d12-3757-11e8-8bbe-4fbd30b6c451.png)

## Steps to reproduce the problem:
Open Postman

**Comment by harryi3t** [comment]
@MBach 
Thanks for reporting this issue. We are looking into this and will get back to you once we have a fix.

**Comment by kalashnikof** [comment]
Same issue here.
Postman 6.4.4
Linux 4.15.0-36-generic / x64

**Comment by ankurdengla** [comment]
@MBach @kalashnikof We looked at this issue and this is happening because while the center option in electron works well for Windows and macOS, it uses the aggregate center of both displays on Linux. This is why the loader window is positioned at the center of both monitors.

This is a known issue on electron - https://github.com/electron/electron/issues/3490

**Comment by ankurdengla** [comment]
The fix for this issue is now available in the latest Canary app - [Postman Canary 7.3.0-canary01](https://www.getpostman.com/downloads/canary). Please check it out and let us know if the issue still persists.

**Comment by kalashnikof** [comment]
@ankurdengla1996 works fine on PostmanCanary-linux-x64-7.3.0-canary01
Linux  5.0.0-15-generic / x64

![Screenshot from 2019-06-18 19-01-06](https://user-images.githubusercontent.com/10325703/59700313-c20d9680-91fb-11e9-95a1-c2826a427be3.png)

**Comment by harryi3t** [comment]
This is fixed in the latest version of Postman (v7.3.4).
